swap_meet/item.py

```diff
@@ -1,2 +1,38 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0, category=""):
+        self.condition = condition
+        self.category = category
+
+        # if no given id, generate uuid
+        if id is None:
+            self.id = uuid.uuid4().int
+        else:
+            self.id = id
+
+
+    def get_category(self):
+        # return class name
+        # return self.__class__.__name__
+        return "Item"
```

Comment:
Writing Python code to dynamically get the class name is preferred. Whenever possible, avoid hardcoding values because hardcoded values cannot be easily inherited across child classes and you lose the power of OOP which enables us to write less code.

Refactoring the method like this will allow you to remove `get_category` from the child classes.

```suggestion
        # return class name
        return self.__class__.__name__
```